includes/ets_user_qa_function.php

```diff
@@ -14,6 +14,9 @@ public function __construct() {
 		// Load The Q & A on click Load More Button
 		add_action( 'wp_ajax_ets_product_qa_load_more',array($this, 'load_more_qa'));
 
+		//shortcode for QA listing
+		add_shortcode( 'display_qa_list',array($this, 'display_qa_listing_shortcode'));
```

Comment:
Move this `add_shortcode` from between the two ajax actions for load_more_qa, and place it after all hooks.